test/built-ins/Symbol/prototype/description/this-val-non-symbol.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-symbol.prototype.description
+description: >
+    Behavior when `this` value is an object without a [[SymbolData]] internal
+    slot.
+info: |
+    1. Let s be the this value.
+    2. Let sym be ? thisSymbolValue(s).
+    3. Return sym.[[Description]].
+features: [Symbol.prototype.description]
+---*/
+
+assert.throws(TypeError, function() {
+  Symbol.prototype.description.call(null);
```

Comment:
This fails, but for the wrong reason. The getter is being called on `Symbol.prototype`. You should instead extract the getter with `Object.getOwnPropertyDescriptor`.